3.10/analyzers.md

```diff
@@ -169,11 +169,14 @@ Currently the following *features* are supported:
 - **position**: enumerate the tokens for position-dependent queries. Required
   for `PHRASE()` and `NGRAM_MATCH()`.
   If present, then the `frequency` feature is also required.
+- **offset**: enable search highlighting capabilities (Enterprise Edition only).
+  Required for `OFFSET_INFO()`.
 
 Also see [PHRASE()](aql/functions-arangosearch.html#phrase),
 [BM25()](aql/functions-arangosearch.html#bm25),
 [TFIDF()](aql/functions-arangosearch.html#tfidf),
-[NGRAM_MATCH()](aql/functions-arangosearch.html#ngram_match).
+[NGRAM_MATCH()](aql/functions-arangosearch.html#ngram_match),
```

Comment:
`NGRAM_MATCH` and `PHRASE` and filtering functions, let's don't mix them up with `OFFSET_INFO`, `BM25`, `TFIDF`.